src/NUnitFramework/framework/Constraints/GreaterThanConstraint.cs

```diff
@@ -21,22 +21,37 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-using System;
-
 namespace NUnit.Framework.Constraints
 {
     /// <summary>
     /// Tests whether a value is greater than the value supplied to its constructor
     /// </summary>
     public class GreaterThanConstraint : ComparisonConstraint
     {
+        /// <summary>
+        /// The Description of what this constraint tests, for
+        /// use in messages and in the ConstraintResult.
+        /// </summary>
```

Comment:
I would recommend not putting XML comments on private members because the end users will never see it.
(Same in other files.)

Reply:
Thanks for pointing it out. 
Why private members in `ComparisonConstraint` are documented then? Not arguing, just trying to understand the documenting process, to avoid such mistakes later on.

Reply:
> Why private members in ComparisonConstraint are documented then?

I'm not sure why that happened. They shouldn't be documented if they are directed at end users. Occasionally an XML summary makes sense on a private member of a large class, but only if it's information directed at users who are editing the class rather than end users.